realm/realm-library/src/androidTest/java/io/realm/TestHelper.java

```diff
@@ -346,11 +346,12 @@ public void log(int level, String tag, Throwable throwable, String message) {
         };
     }
 
+    // Generate a random string with only capital letters which is always a valid class/field name.
     public static String getRandomString(int length) {
         Random r = new Random();
         StringBuilder sb = new StringBuilder(length);
         for (int i = 0; i < length; i++) {
-            sb.append((char) r.nextInt(128)); // Restrict to standard ASCII chars.
+            sb.append((char) r.nextInt(25) + 'A'); // Restrict to capital letters
```

Comment:
`r.nextInt(25)` should be `r.nextInt(26)` or `r.nextInt('Z' + 1 - 'A')`.

`r.nextInt(x)` returns a value between `0` to `x - 1`.